inventory/hosts.example

```diff
@@ -783,6 +783,10 @@ debug_level=2
 #
 # Detected names may be overridden by specifying the "names" key
 #openshift_master_named_certificates=[{"certfile": "/path/to/custom1.crt", "keyfile": "/path/to/custom1.key", "names": ["public-master-host.com"], "cafile": "/path/to/custom-ca1.crt"}]
+#
+# Add a trusted CA to all pods, copies from the control host, may be multiple
```

Comment:
Make this a list instead of a single item to support multiple ones?  That way we don't have to revise in the future.  Also, could be more clear that the /path/to/file is actually on the control host and not the remote host.

Reply:
I think it's ok in this case to keep it as a single item since you can add multiple certs to a single file.